CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ This project adheres to [Semantic Versioning](http://semver.org/) starting from
 ## Table of contents
 
 #### 4.x Releases
-- `4.2.0` Releases - [4.2.0](#420) | [4.2.1](#421) | [4.2.2](#422) | [4.2.3](#423) | [4.2.4](#424)
+- `4.2.0` Releases - [4.2.0](#420) | [4.2.1](#421) | [4.2.2](#422) | [4.2.3](#423) | [4.2.4](#424) | [4.2.5](#425)
```

Comment:
Thanks for updating those! Are the blocks down below in the readme (line 43+) updated automatically, or would we need to add them manually?